help/destinations/catalog/email-marketing/oracle-eloqua-api.md

```diff
@@ -0,0 +1,165 @@
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+![Platform UI screenshot showing the destination details.](../../assets/catalog/email-marketing/oracle-eloqua-api/destination-details.png)
+
+* **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+* **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Oracle Eloqua] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL Oracle Eloqua] destination fields, follow these steps:
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
+    * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Oracle Eloqua] instance:
+        |Source Field|Target Field| Mandatory|
```

Comment:
Some clarification questions: 

1. Are we absolutely sure that **both** these fields (`Attribute: emailAddress` and `Identity: EloquaId`) are absolutely necessary? What happens if a user doesn't select one or both of them?
2. Do the mappings need to be exactly as described in the table? Namely, on the Source Field side, can users select other attributes/identities and map them to the Target fields?

Reply:
_Are we absolutely sure that both these fields (Attribute: emailAddress and Identity: EloquaId) are absolutely necessary? What happens if a user doesn't select one or both of them?_

Dev team : Yes ,EloquaID is needed in the http call as corresponding to that all other attributes are updated, Email Id is also necessary as without it the API throws error.

```
    {
        "type": "ObjectValidationError",
        "container": {
            "type": "ObjectKey",
            "objectType": "Contact"
        },
        "property": "emailAddress",
        "requirement": {
            "type": "EmailAddressRequirement"
        },
        "value": "<null>"
    }
]  
```

_Do the mappings need to be exactly as described in the table? Namely, on the Source Field side, can users select other attributes/identities and map them to the Target fields?_

No, All we need to consider is the target mapping. The attributes need to be named as described in table because these attributes will form request body.
From source side we don't have any restriction to map fields ,you can map from any where you want if the data is appropriate it will be pushed to destination else will throw an error.
(Example - you can map EloquaID to any identity namespace like contact key,outreach ID etc. but if that ID is not present in Eloqua DB it will throw an error.

Reply:
These are great clarifications, thanks!

Could we transform these clarifications into material that we could add to this documentation page?

Reply:
Sure, incorporated.